kube-insight-installation-in-GKE.md

```diff
@@ -0,0 +1,196 @@
+NAME       LOCATION         MASTER_VERSION  MASTER_IP      MACHINE_TYPE   NODE_VERSION  NUM_NODES  STATUS
+insight    europe-north1-a  1.9.7-gke.6     35.228.164.55  n1-standard-1  1.9.7-gke.6   3          RUNNING
+monitored  europe-north1-a  1.9.7-gke.6     35.228.38.59   n1-standard-1  1.9.7-gke.6   3          RUNNINGli_wu@cloudshell:~ (test1012-219213)$
+```
+### 1.4. Enable kubectl and get cluster-info with --cluster specified
+```
+kubectl cluster-info --cluster gke_test1012-219213_europe-north1-a_insight
+kubectl cluster-info --cluster gke_test1012-219213_europe-north1-a_monitored
+```
+## 2. Set up a virtual environment for manifestr
+### 2.1  install pipenv
+```
+sudo pip install pipenv
+```
+### 2.2 setup manifestr
+```
+pipenv install --three
+pipenv shell
+```
+## 3. Setup monitored cluster
+
+### 3.1 Deploy metrics
+#### 3.1.1 Render j2 files
+```
+(manifestr)li_wu@cloudshell:~/kube-insight-manifests/manifests/agents/metrics (test1012-219213)$ manifestr --values `pwd`/values.py --template-root-dir `pwd`/templates --output-dir `pwd`/output
+overwrite /home/li_wu/kube-insight-manifests/manifests/agents/metrics/output (y/n)?y
+```
+#### 3.1.2 Create namespace metrics
+```
+(manifestr)li_wu@cloudshell:~/kube-insight-manifests/manifests/agents/metrics (test1012-219213)$ kubectl create ns metrics --cluster gke_test1012-219213_europe-north1-a_monitored
+namespace "metrics" created
+```
+#### 3.1.3. Set user with cluster-admin for deploy RBAC
+```
+kubectl create clusterrolebinding cluster-admin-binding   --clusterrole cluster-admin   --user $(gcloud config get-value account) --cluster gke_test1012-219213_europe-north1-a_monitored
+```
+
+Note: if this step is missing, you will get following error during step 3.1.4
+```
+Error from server (Forbidden): error when creating "output/node-exporter/node-exporter-rbac.yaml": clusterroles.rbac.authorization.k8s.io "node-exporter" is forbidden: attempt to grant extra privileges: [PolicyRule{Resources:["tokenreviews"], APIGroups:["authentication.k8s.io"], Verbs:["create"]} PolicyRule{Resources:["subjectaccessreviews"], APIGroups:["authorization.k8s.io"], Verbs:["create"]}] user=&{li.wu@elastisys.com  [system:authenticated] map[user-assertion.cloud.google.com:[AGKDXmrmxR4ldy6MKeKjP/yU11f3oYRk8PD+N0PgGAq22xLez1St1GzcuML16US1LpSUuGhBOnxyxwZsHp7SCQjDpznRqruCdKWr/VWWNPbAwug79cP3/aWm082ChAydsV4O9mpVKPNRdUeQGWHrVs1nNYADq9+a39CgHjH6lHF0hGtRHFdZ+21irqPHQYoc3/FkERUC2ax9wRdypoU/lgYoRVj9o4siTnwHcmU=]]} ownerrules=[PolicyRule{Resources:["selfsubjectaccessreviews" "selfsubjectrulesreviews"], APIGroups:["authorization.k8s.io"], Verbs:["create"]} PolicyRule{NonResourceURLs:["/api" "/api/*" "/apis" "/apis/*" "/healthz" "/swagger-2.0.0.pb-v1" "/swagger.json" "/swaggerapi" "/swaggerapi/*" "/version"], Verbs:["get"]}] ruleResolutionErrors=[]
+```
+#### 3.1.4 Deployment
+`kubectl apply -R -f output/ --cluster gke_test1012-219213_europe-north1-a_monitored`
```

Comment:
Somehow this snippet gets merged into the one under? Do you need three ` here as well?